scripts/update_env.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+
+from yaml import dump, load
+
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("file", help="YAML file path")
+    parser.add_argument(
+        "-p",
+        "--packages",
+        help="File containing packages requirements " "to add to YAML file.",
+    )
+    parser.add_argument(
+        "-o", "--output", help="Output YAML file. By default, print output."
+    )
+    args = parser.parse_args()
+    with open(args.file) as file:
+        data = load(file, Loader=Loader)
+    if args.packages:
+        packages = []
+        with open(args.packages) as file:
+            for line in file:
+                line = line.strip()
+                if not line:
+                    continue
+                packages.append(line)
+        dependencies = data.get("dependencies", [])
+        packages_added = False
+        for index, element in enumerate(dependencies):
+            if isinstance(element, dict):
+                dependencies = (
+                    dependencies[:index] + packages + dependencies[index:]
+                )
+                packages_added = True
+                break
+        if not packages_added:
+            dependencies.extend(packages)
+        data["dependencies"] = dependencies
+    data.pop("name", None)
+    output = dump(data, Dumper=Dumper)
+    if args.output:
+        with open(args.output, "w") as file:
+            file.write(output)
+    else:
+        print(output)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
It might be nice to not have three copies of this script around.

Reply:
I can create a special package, something like `myia_utils`, and move this file into it.

Reply:
Since all the packages are within the myia repo for now, we could just reuse the script.  We can revisit this if we split the repos.

Reply:
OK ! I already moved the script into a package `myia_utils` in previous commit. Should I just move it back to a simple script in myia `scripts` folder ?